front-end/src/components/KeyMeasureDiagnostic/index.vue

```diff
@@ -31,10 +31,10 @@
       QualityMeasureDiagnostic,
       WasteMeasureDiagnostic
     },
-    props: ['measure', 'closeModal'],
+    props: ['measure', 'closeModal', 'originalDiagnostic'],
     data() {
       return {
-        diagnostic: diagnostics[this.measure.id] || {},
+        diagnostic: this.originalDiagnostic,
```

Comment:
On peut directement mettre `diagnostic` en props plutôt ici.

Reply:
Comment est-ce que vue peut distinguer entre le prop et le data quand on fait `this.diagnostic` ?

Reply:
Il peut pas, si y'a déjà `diagnostic ` en props il ne peut pas y avoir `diagnostic ` dans data. Qu'on déclare `diagnostic ` dans props, data ou même computed, il ne peut y en avoir qu'un et c'est celui-là que renverra `this.diagnostic `